sdk/communication/azure-communication-chat/src/test/java/com/azure/android/communication/chat/implementation/notification/NotificationUtilsTest.java

```diff
@@ -59,6 +59,30 @@ public void canParseMessageMetadata() {
         assertEquals(metadataMap.get("tags"), "tag1");
     }
 
+    @Test
+    public void canParseTokenForRegion() {
+        final String usaSkypeToken = "eyJhbGciOiJSUzI1NiIsImtpZCI6IjY5RUU4QzJFNjUwNzI1OUJBMEYxREQ3MDU0Njk0QjdDQTI4QkQxQkYiLCJ4NXQiOiJhZTZNTG1VSEpadWc4ZDF3VkdsTGZLS0wwYjgiLCJ0eXAiOiJKV1QifQ.eyJza3lwZWlkIjoiYWNzOjE4OGQ0Y2VhLTBhOWItNDg0MC04ZmRjLTdhNWM3MWZlOWJkMF8wMDAwMDAxYS1kZDM2LTE0OTktNzdkNC0wMjQ4MjIwMDA1YmUiLCJzY3AiOjE3OTIsImNzaSI6IjE2OTU2NzM3NjYiLCJleHAiOjE2OTU3NjAxNjYsInJnbiI6ImFtZXIiLCJhY3NTY29wZSI6ImNoYXQiLCJyZXNvdXJjZUlkIjoiMTg4ZDRjZWEtMGE5Yi00ODQwLThmZGMtN2E1YzcxZmU5YmQwIiwicmVzb3VyY2VMb2NhdGlvbiI6InVuaXRlZHN0YXRlcyIsImlhdCI6MTY5NTY3Mzc2Nn0.OmjphUp-TEze3R4a1mo0puUdn7bYw-rFWheodWoe5H-uwKQkTQ_9BzreUJ4qefrXinNlgyVz7DdHAp-_fyh5EQyN6CTwpZdvE4qqmFexk7LJC53GGtGPJVRM_JIXN6vKMDFFrR6wWqwZ22Urr0LpvB2xQRQJJmd9uSOj5TfilJOZrgGTGC0LuKTGLC6kEsAtbDod7zzclZWiApnsG7hgClx86C3fa7-WxLrR_gvqD77HUQOZoNu13uAAVkpFpC1Dk78IhuJfu_m_yEJK0D0z80y0yWN-WE0WLjfnaJNCk8aFbfJdEmvRBQP_mBHBKMVoRDwVYkvVGcXdDYnankYgCw";
```

Comment:
Are we able to base64 encode on the fly? For example:
```
String fakeValue = "{\"iss\":\"ACS\",\"iat\": 1608152725,\"exp\": 1739688725,\"aud\": \"\",\"sub\": \"\",\"resourceLocation\": \"unitedstates\"}";
String usaSkypeToken = (
            new StringBuilder(a)
                .append("eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.")
                .append(Base64.getEncoder().encodeToString(fakeValue.getBytes()))
                .append(".EMS0ExXqRuobm34WKJE8mAfZ7KppU5kEHl0OFdyree8")).toString();
```
That way we know at a glance that the token contains the property `resourceLocation` and its value is set to `unitedstates`